framework/web/UrlRule.php

```diff
@@ -232,8 +270,11 @@ public function parseRequest($manager, $request)
             return false;
         }
 
-        $pathInfo = $request->getPathInfo();
         $suffix = (string)($this->suffix === null ? $manager->suffix : $this->suffix);
+        $pathInfo = $request->getPathInfo();
+        if ($this->hasNormalizer($manager)) {
+            $pathInfo = $this->getNormalizer($manager)->normalizePathInfo($pathInfo, $suffix, $normalized);
```

Comment:
Maybe, it's better to initialize `$normalize` as `false`, before passing it somewhere?